deployment-aws/puppet/mevapi/manifests/init.pp

```diff
@@ -12,8 +12,7 @@
   String                  $backend_domain,
   String                  $cloudwatch_log_group,
   Optional[String]        $container_registry = 'github',
-  String                  $cromwell_bucket_name,
-  String                  $cromwell_server_ip,
+  String                  $nextflow_bucket_name,
```

Comment:
Should this be optional?

Reply:
technically we don't NEED a remote job runner (i.e. one that executes jobs elsewhere like on AWS Batch), so yes, it could be options. Tradeoff is that if we make this optional, we'd need to perform some additional checks at startup to ensure that there's an existing bucket in the case where remote job runners are enabled/desired. I'm inclined to leave it since even an empty bucket isn't really a problem per se.